templates/codelists/version.html

```diff
@@ -38,6 +38,17 @@ <h3>{{ codelist.name }}</h3>
 
     {% if user.is_authenticated %}
     <hr />
+
+    <div class="btn-group-vertical btn-block" role="group">
+      <a
+        class="btn btn-outline-info btn-block"
+        href="{% url 'codelists:codelist-edit' project_slug=codelist.project.slug codelist_slug=codelist.slug %}">
+        Edit
```

Comment:
Could this be "Edit codelist metadata"?

Reply:
I went with `Edit metadata` since the longer form overlapped the button boundary:

![](https://p198.p4.n0.cdn.getcloudapp.com/items/WnuJNgl2/Image%202020-08-11%20at%202.46.08%20pm.png?v=ae13ecb5762fcc78681c8ed9acfe8455)

Fixed in 1912ea1